DFReader.py

```diff
@@ -382,17 +382,32 @@ def dump_verbose(self, f):
                     f.write("    %s: %s" % (c, val))
                 except UnicodeDecodeError:
                     f.write("    %s: %s" % (c, to_string(val)))
+
+            # see if this is an enumeration entry, emit enumeration
+            # entry name if it is
+            if c in field_metadata_by_name:
+                fm = field_metadata_by_name[c]
+                fm_enum = getattr(fm, "enum", None)
+                if fm_enum is not None:
+                    enum_entry_name = "?????"  # default, "not found" value
+                    for entry in fm_enum.iterchildren():
+                        if int(entry.value) == int(val):
+                            enum_entry_name = entry.get('name')
+                            break
+
+                    f.write(f" {enum_entry_name}")
```

Comment:
```suggestion
                    f.write(f" ({enum_entry_name})")
```
Maybe brackets around the enum entry name for consistency with what is done for tlogs.

Reply:
Cool.  I was going backwards and forwards on this, so I'll make the change.